clang/test/CodeGen/aarch64-varargs.c

```diff
@@ -834,5 +834,4 @@ typedef struct {} empty;
 empty empty_record_test(void) {
 // CHECK-LABEL: define{{.*}} void @empty_record_test()
   return va_arg(the_list, empty);
-// CHECK: [[GR_OFFS:%[a-z_0-9]+]] = load ptr, ptr @the_list
```

Comment:
Maybe should check something here?  Something like `// CHECK: call void @llvm.va_start  // CHECK-NEXT: ret void`.

Reply:
Well, there is no va_start but I checked ret